app/components/loading-indicator.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const LoadingIndicator = ({ children, off }) => {
+  const visibility = off ? 'hidden' : '';
```

Comment:
I think this should strictly be `const visibility = off ? 'hidden' : 'visible';` but it's not a big deal.